### ISSUE TITLE
Error trying to persist (sometimes) using localStorage

### ISSUE THREAD

**Issue by vjrj** [plrenaudin/svelte-storez#6]
Sometimes, when I try to persist calling dispose() I get this error:
```
index.mjs:58 Uncaught (in promise) TypeError: stop is not a function
    at index.mjs:58
    at storez.js:124
```

I think that is related with the content of my store in some situations.

I can reproduce the error even more if I call dispose twice:
```
dispose();
dispose();
```

**Comment by plrenaudin** [comment]
Hi @vjrj thanks for reporting this. Can you try the latest version as it might have fixed that (0.3.9)?

**Comment by vjrj** [comment]
Yest, it seems is fixed in 0.3.9.

Thanks indeed @plrenaudin
